backend/src/api/base.clj

```diff
@@ -1,24 +1,22 @@
 (ns api.base
   (:require [rules.create-game :as create-game]
             [rules.play-card :as play-card]
-            [rules.victory-conditions :as victory-conditions]
             [persistence.persistence :as persistence]
-            [api.helper :as helper]
+            [api.player-view :as player-view]
+            [api.player-view-functions :as functions]
```

Comment:
I think `api.player-view-functions` should only be used from inside `api.player-view`...

Reply:
where should I put `player-num`, then? :(

Reply:
maybe move `player-num` and `translate-player` to `api.conversions` or something?